website/docs/support/survey.md

```diff
@@ -0,0 +1,10 @@
+---
+id: survey
+title: Support Customer Survey
+---
+
+# Thank you
+
+We know how valuable is your time and really appreciate the feedback on our work.
+<br></br>
+Please answer the two questions below and reach us out if you want to add anything else.
```

Comment:
What questions? Am I missing something?

Reply:
It is overlayed by the analytics plugin later. The page is simply a trigger.